### PR TITLE
TVS: cleanup

### PR DESCRIPTION
**Pull request by snazy**
Big-bang-cleanup PR

---
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/2112)

### CODE REVIEW

**Comment by snazy** [comment]
Note: there are two commits: one that "only" deletes a bunch of files and one commit with changes to files.

File: site/docs/try/configuration.md [hunk 1]
```diff
@@ -15,20 +15,11 @@ docker run -p 8080:8080 projectnessie/nessie \
...
-## Dynamo version store specific configuration
-# should Nessie create its own dynamo tables
-nessie.version.store.dynamo.initialize=false
-
-## Dynamo Configuration
-quarkus.dynamodb.aws.region=us-west-2
-quarkus.dynamodb.aws.credentials.type=DEFAULT
-# quarkus.dynamodb.endpoint-override=http://localhost:8000
+# path if using RocksDB
+nessie.version.store.rocks.directory=/tmp/rocks-nessie
```
Comment (dimas-b) [inline site/docs/try/configuration.md:22]: Isn't this effectively the same as "in memory"? Should we suggest a non-transient location perhaps?

File: versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java [hunk 2]
```diff
@@ -89,24 +89,22 @@ public String getId(String contents) {
...
+    return 0;
   }
 
   @Override
   public TestEnum getType(Byte payload) {
-    return getValueSerializer().getType(payload);
+    if (payload == null) {
+      return StringStoreWorker.TestEnum.NULL;
+    }
+    return payload > 60 ? StringStoreWorker.TestEnum.YES : StringStoreWorker.TestEnum.NO;
```
Comment (dimas-b) [inline versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java:100]: why 60 and not 42 ;-)

**Review by dimas-b — COMMENTED** [review]
Reviewed the second commit only :)

File: versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java (hunk 2 above)
Comment (snazy) [reply versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java:100]: IDK - was like that before. It's a test-only thing though ;)

Comment (snazy) [reply versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java:100]: I'd have DEFINITELY chosen 42 :P

File: site/docs/try/configuration.md (hunk 1 above)
Comment (snazy) [reply site/docs/try/configuration.md:22]: `/tmp/` has the advantage of being accessible to all users (unlike `/var/lib` or so).

Note: `/tmp` is not always transient in Linux though. [FHS 3.0](https://refspecs.linuxbase.org/FHS_3.0/fhs/ch03s18.html) states that you should not _assume_ that files survive reboots, but at least Ubuntu does not clear `/tmp` nor does it periodically do that (anymore). I suspect, clearing `/tmp`just caused too many problems (a lot of things place important, long-living things there - e.g. Java's hsperfdata, systemd, X11, etc).

Comment (dimas-b) [reply site/docs/try/configuration.md:22]: fair enough.

**Review by dimas-b — DISMISSED** [review]

**Review by nastra — APPROVED** [review]

**Pull request merged**